packages/approval-controller/src/ApprovalController.test.ts

```diff
@@ -109,6 +118,7 @@ describe('approval controller', () => {
           id: 'foo',
           origin: 'bar.baz',
           requestData: null,
+          requestState: undefined,
```

Comment:
```suggestion
```

Reply:
I can't remove it otherwise the test fails

Reply:
Ah, this is fine. I thought this was the parameters to `add()`, it is not. It is just the state values.